src/v2/guide/migration.md

```diff
@@ -1257,57 +1257,65 @@ Los enlaces _one-time_ (`{% raw %}{{* foo }}{% endraw %}`) han sido reemplazados
 </div>
 {% endraw %}
 
-## Reactivity
+## Reactividad
 
-### `vm.$watch` <sup>changed</sup>
+### `vm.$watch` <sup>cambiado</sup>
 
-Watchers created via `vm.$watch` are now fired before the associated component rerenders. This gives you the chance to further update state before the component rerender, thus avoiding unnecessary updates. For example, you can watch a component prop and update the component's own data when the prop changes.
+Los _watchers_ creados a través de `vm.$watch` ahora se disparan antes de que el componente asociado se vuelva a renderizar. Esto le brinda la oportunidad de actualizar el estado antes de que el componente vuelva a renderizarse, evitando así actualizaciones innecesarias. Por ejemplo, puede escuchar por una propiedad y actualizar sus propios datos cuando la propiedad cambia.
 
-If you were previously relying on `vm.$watch` to do something with the DOM after a component updates, you can instead do so in the `updated` lifecycle hook.
+Si anteriormente usted dependía de `vm.$watch` para hacer algo con el DOM después de las actualizaciones de un componente, usted puede hacerlo en el enlace del ciclo de vida `updated`.
 
 {% raw %}
 <div class="upgrade-path">
-  <h4>Upgrade Path</h4>
-  <p>Run your end-to-end test suite, if you have one. The <strong>failed tests</strong> should alert to you to the fact that a watcher was relying on the old behavior.</p>
+  <h4>Ruta de actualización</h4>
+  <p>
+    Ejecute su conjunto de pruebas de extremo a extremo, si tiene uno. Las <strong>pruebas que fallen</strong> les deberían de alertar sobre el hecho que un _watcher_ dependía de un comportamiento anterior.
```

Comment:
🤔 
```suggestion
    Ejecute su conjunto de pruebas de extremo a extremo, si tiene uno. Las <strong>pruebas que fallen</strong> les deberían de alertar sobre el hecho de que un _watcher_ dependía de un comportamiento anterior.
```

Reply:
Also _watcher_ won't make that text to be in italics (as this is inside a `{% raw %}` instead of plain markdown. You need to use `<i>` tags:

```suggestion
    Ejecute su conjunto de pruebas de extremo a extremo, si tiene uno. Las <strong>pruebas que fallen</strong> les deberían de alertar sobre el hecho que un <i>watcher</i> dependía de un comportamiento anterior.
```

Reply:
You are right on this one! thanks for that.

Regarding the `de que` I'm pretty sure adding that is not the right way to right it. Also it sounds familiar to me that that is something called "de queismo" and is not correct.

But I'm not 100% sure.

Reply:
Yeah, I agree with that, I wasn't really sure 🤷‍♂